src/arguments.h

```diff
@@ -71,6 +71,7 @@ class Arguments {
     Counter _counter;
     const char* _event;
     long _interval;
+    long _jstackdepth;
```

Comment:
`int` is definitely enough for this. I wonder how this compiles without a warning when you later assign it to `int max_depth`.